frontend/src/pages/ModelRegister/utils/schema.json

```diff
@@ -0,0 +1,322 @@
+{
+  "type": "object",
+  "properties": {
+    "NO_NAME_FIELD_$0": {
+      "name": "NO_NAME_FIELD_$0",
+      "x-component": "CustonHeader",
+      "x-component-props": {
+        "title": "Model Registeration"
```

Comment:
Maybe you should put this as the page header, rather than a component of the form
https://ant.design/components/page-header/

![image](https://user-images.githubusercontent.com/36268431/111060924-5b560380-84db-11eb-83f9-1aefd12c3641.png)

And put the right corner buttons in this order: [Cancel] [Upload]

Reply:
Because it's convenient to put buttons inside the form to handle form event, this component customized from `page-header`  so the style is the same